exodus/search/templates/search_form.html

```diff
@@ -1,107 +1,165 @@
-                       placeholder="fr.meteo">
-                    <div class="input-group-append">
-                        <button id="go" class="btn btn-secondary" type="button">Go!</button>
-                    </div>
-                </div>
-
-                <small id="query_help" class="form-text text-muted text-center">{% trans "Here you can search for an application using its name or handle" %}
-                    <i>i.e.</i> <code><b>fr.meteo</b></code><br>
-                </small>
-                <br>
-
-                <h2 style="display: none" id="results-title">{% trans "Results" %}</h2>
-                <div id="results" style="display: none" class="">
-                </div>
-
-            </div>
+<div class="row justify-content-center">
+  <div class="col-xl-3 col-lg-3 col-md-3 col-sm-4 col-4 d-flex align-items-center">
+    <img src="/static/img/logo_purple_small.svg" class="mx-auto d-block img-fluid">
+  </div>
+  <div class="col-xl-5 col-lg-5 col-md-8 col-sm-8 text-center">
+    <h1 class="main-title">εxodus</h1>
+    <h5><b>{% trans "The privacy audit platform for Android applications" %}</b></h5>
+  </div>
+</div>
+<br>
+<div class="row justify-content-center">
+  <div class="col-lg-8 col-sm-12">
+    <div class="form-group">
+      {% csrf_token %}
+      <h4>{% trans "Search a report" %}</h4>
+      <label for="query">{% trans "Application name" %}</label>
+      <div class="input-group input-group-lg mb-3">
+        <input type="text" class="form-control" id="query" aria-describedby="query_help" placeholder="fr.meteo">
+        <div class="input-group-append">
+          <span class="input-group-text" id="query_help">
+            <small><img src="/static/img/search.svg"></img></small>
+          </span>
+        </div>
+      </div>
+      <small class="form-text text-muted text-center">{% trans "Here you can search for an application using its name or handle" %}
+        <i>i.e.</i> <b>fr.meteo</b><br>
+      </small>
+      <br>
+      <div id="go-div" class="row justify-content-center">
+        <div class="col-6 col-md-4">
+          <button id="go" class="btn btn-secondary btn-block">{% trans "Let's go!" %}</button>
+        </div>
+      </div>
+      <h4 style="display: none" id="results-title">{% trans "Results" %}</h4>
+      <div id="results" style="display: none">
+      </div>
     </div>
+  </div>
+</div>
+<br>
+<br>
+<br>
+<div class="row justify-content-center">
+  <div class="col-lg-8 col-sm-12">
+    {% trans "εxodus analyzes Android applications in order to <b>list the embedded trackers.</b>" %}
+    <br>
+    <br>
+    {% trans "A tracker is a piece of software <b>meant to collect data about you or your usages.</b> So, εxodus reports tell you what are the ingredients of the cake." %}
+    <br>
+    <br>
+    {% trans "εxodus does not decompile applications, its analysis technic is legal." %}
+  </div>
+</div>
+<br>
```

Comment:
Did you know if it's better to use `<br>` than margins?

Reply:
Margins are probably the way to go, that was mostly laziness from my side :). I'll take a look to change that, thanks!